edxval/api.py

```diff
@@ -280,6 +293,28 @@ def get_video_transcript_url(video_id, language_code):
         return video_transcript.url()
 
 
+def create_video_transcript(video_id, language_code, file_format, content, provider=TranscriptProviderType.CUSTOM):
```

Comment:
Where are we using this method?

Reply:
Why are we not using `create_or_update_video_transcript` method?

Reply:
This is required for migration task. we cannot use `create_or_update` as it also overwrites the existing transcript.